go.mod

```diff
@@ -0,0 +1,51 @@
+module github.com/percona/pmm-managed
+
+go 1.16
+
```

Comment:
Add please comments similar to https://github.com/percona-platform/template/blob/4b2f2acce3592549eb136486c6720ae71022e0a9/go.mod#L5-L9

for `saas` and `pmm` dependencies, please

Reply:
We can't use replace directive because pmm is not migrated to go modules

Reply:
Ok, then only for `saas` for now